ibm/resource_ibm_is_subnet_reserved_ip.go

```diff
@@ -167,6 +179,10 @@ func resourceIBMISReservedIPRead(d *schema.ResourceData, meta interface{}) error
 		d.Set(isReservedIPName, *rip.Name)
 		d.Set(isReservedIPOwner, *rip.Owner)
 		d.Set(isReservedIPType, *rip.ResourceType)
+		target, ok := rip.Target.(*vpcv1.ReservedIPTarget)
```

Comment:
Can you first add a check if rip.Target != nil {

---------
}
above files also